### DISCUSSION TITLE
I cant see the "new Task"-Space on my second screen

### DISCUSSION THREAD

**Discussion by Lesekater** [Hanziness/AnotherPomodoro#175] [Ideas & feature requests]
Hi, I just started using the app and was a bit confused when trying to create a new task. Admittedly my second screen is quite low contrast, but I think it would be better to highlight the area with a "New Task" placeholder or change the color a bit.

![image](https://user-images.githubusercontent.com/22131220/152680332-f4a9eb80-41d1-449c-97d1-97157977ea16.png)

**Comment by Hanziness** [comment]
Hi @Lesekater! Thank you for mentioning this, adding at least a placeholder is a very good idea. I'll also look into the contrast with the text area!
